src/modules/todos/controllers.js

```diff
@@ -0,0 +1,89 @@
+import { todos, users } from '../database/database.js';
+import AppException from '../../utils/exceptions/AppException.js';
+
+export function getTodos(_, res) {
+	res.json(todos);
+};
+
+export function createTodo(req, res, next) {
+	try {
+		const title = req.body.title;
+		const status = req.body.status;
+		const date = req.body.date;
+		const todo = {
+			id: req.userId,
```

Comment:
looks to me like you are setting the id of the todo to the user id. This seems off. How then would we uniquely identify a todo? May I suggest this instead?
```
{
    id: <something unique for each todo>,
    status: <an enumeration of possible status (pending, started, completed). should always start at pending>,
    title: <the name of the todo>,
    date: <the deadline>,
    userId: <the id of the owner of this todo>,
    createdAt: <the date the todo was created>,
    updatedAt: <the last time the todo was updated>,
}
```